shootout/shootout.py

```diff
@@ -142,32 +152,132 @@ async def soset_bet(self, ctx, amount: int):
         await ctx.tick()
         return
 
-    @soset.command(name="message")
-    async def soset_message(self, ctx, message: str):
-        """Sets the message that the bot listens for during shootout games. Only affects new games!"""
-        await self.config.guild(ctx.guild).message.set(message)
+    @soset.command(name="draw")
+    async def soset_draw(self, ctx, *, draws: str):
```

Comment:
If no draws are given as an argument, this should list all draws currently set. You can do this by adding a default value of None to draws argument and checking if draws is None in function.

Reply:
Implemented.